Source/Particles/WarpXParticleContainer.H

```diff
@@ -280,7 +281,7 @@ public:
 
     virtual void ConvertUnits (ConvertDirection convert_dir){};
 
-    static void ReadParameters ();
```

Comment:
Would can't this remain `static`?

Reply:
I had changed it initially when reading the parser in this function. Now since the functions are parsed in MultiParticleContainer::ReadParameters(), this should be changed back to static! Thank you